homeassistant/components/sensor/rest.py

```diff
@@ -68,20 +72,25 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     rest = RestData(method, resource, auth, headers, payload, verify_ssl)
     rest.update()
 
-    add_devices([RestSensor(hass, rest, name, unit, value_template)], True)
+    add_devices([RestSensor(hass, rest, name, unit,
+                            value_template, json_attrs)])
 
 
 class RestSensor(Entity):
     """Implementation of a REST sensor."""
 
-    def __init__(self, hass, rest, name, unit_of_measurement, value_template):
+    def __init__(self, hass, rest, name,
+                 unit_of_measurement, value_template, json_attrs):
         """Initialize the REST sensor."""
         self._hass = hass
         self.rest = rest
         self._name = name
         self._state = STATE_UNKNOWN
         self._unit_of_measurement = unit_of_measurement
         self._value_template = value_template
+        self._json_attrs = json_attrs
+        self._attributes = None
+        self.update()
```

Comment:
Remove this

Reply:
When you say "this", presumably you mean the `self.update()` rather than the whole section.